crates/rerun_c/src/rerun.h

```diff
@@ -131,22 +132,17 @@ typedef struct rr_store_info {
     rr_store_kind store_kind;
 } rr_store_info;
 
-/// Arrow-encoded data of a single component for a single entity.
+/// Arrow-encoded data of a single batch components for a single entity.
 typedef struct rr_data_cell {
     /// The name of the component, e.g. `position`.
     rr_string component_name;
 
-    /// The number of bytes in the `bytes` field.
-    /// Must be a multiple of 8.
-    uint64_t num_bytes;
+    /// A batch of components serialized into an arrow array.
+    ArrowArray array;
 
-    /// Data in the Arrow IPC encapsulated message format.
-    ///
-    /// There must be exactly one chunk of data.
-    ///
-    /// * <https://arrow.apache.org/docs/format/Columnar.html#format-ipc>
-    /// * <https://wesm.github.io/arrow-site-test/format/IPC.html#encapsulated-message-format>
-    const uint8_t* bytes;
+    /// The schema used for the arrow array.
+    /// TODO(andreas): Use a schema registry that identifies this and the component name with a unique schema ID.
+    ArrowSchema schema;
```

Comment:
Can't we just statically allocate the schema and always pass the same pointer to it in here? Why do we need a registry?

Reply:
see comment below: I want to also remove the Rust sided cost of FFI schema parsing & building. Also practically speaking we'd still need to allocate it somewhere since we don't get this schema allocated by libarrow, it only fills out a struct (which in turn holds on to allocs)